linux/vhba_hot_add_remove/hot_add_remove_disk.yml

```diff
@@ -36,9 +36,15 @@
     disk_size_gb: "{{ new_disk_size_gb }}"
     ctrl_number: "{{ new_ctrl_number }}"
     unit_number: "{{ new_unit_number }}"
-  when: not add_new_controller
+  when: new_disk_ctrl_state == "existing"
 
-- name: "Wait for hot-added disk being present in guest OS"
+- name: "Collect dmesg after hot adding a disk to {{ new_disk_ctrl_state }} {{ new_disk_ctrl_type }} controller"
+  include_tasks: ../utils/collect_dmesg.yml
+  vars:
+    dmesg_options: "-c"
+    dmesg_output_file_name: "dmesg_after_{{ new_disk_ctrl_state }}_ctrl_disk_hot_add.log"
```

Comment:
Here no need to add `dmesg_no_log: true`?

Reply:
No, it doesn't. The dmesg will be output to the file, and no need to output it to console.